test/testrunner.py

```diff
@@ -43,8 +43,10 @@ def run_sagemaker_local_tests(images):
     framework, _ = get_framework_and_version_from_tag(images[0])
     sm_tests_path = os.path.join("test", "sagemaker_tests", framework)
     sm_tests_tar_name = "sagemaker_tests.tar.gz"
-    run(f"tar -cz --exclude='*.pytest_cache' --exclude='__pycache__' -f {sm_tests_tar_name} {sm_tests_path}")
-    ec2_client = boto3.client("ec2", config=Config(retries={"max_attempts": 10}), region_name=DEFAULT_REGION)
+    run(
```

Comment:
can you run `black -l 120` on this file or revert the formatting changes (we use black formatting tool, can be installed via `pip install black`)?